components/datatypes/data/refunditem.schema.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Please move this to the one folder above 
.../components/datatypes/refunditem.schema.json
All the example files will also move along.

Reply:
@mjasrotia done. i had it in the same directory as `payment` which i see as kind of the inverse/equivalent of it. should we move `payment` out too?

Reply:
I can move them later. Few old data types are in that directory and somehow hardcoded in the code so, can't move without testing. Any new ones, we tend to create under the datatype folder itself.